.gitignore

```diff
@@ -14,3 +14,7 @@ pom.xml.releaseBackup
 .idea
 *.sublime-project
 *.sublime-workspace
+
+# vim
+*.swp
+Session.vim
```

Comment:
I am lost as to how you want me to do with this change. Since you want me to remove this change, I will send a commit that removes it, then another commit that puts it back? Or you want this change on a different pull request?